BrainPortal/app/views/bourreaux/show.html.erb

```diff
@@ -362,6 +362,7 @@
           <%= text_field_tag "bourreau[docker_executable_name]", @bourreau.docker_executable_name, :size => 60 %><br/>
            <div class="field_explanation">Name of the Docker executable available on the machines where tasks will run. Defaults to "docker" if left empty.</div>
         <% end %>
+        <% t.boolean_edit_cell( 'bourreau[docker_present]', !!@bourreau.docker_present, true, false, :header => "Docker Present?" ) %>
```

Comment:
If I recall correctly, it can be nil, but it needs to be false in that case (i.e. boolean type in this case) or it does not work (i.e. a `value==false` is done somewhere, which nil does not satisfy).